### ISSUE TITLE
[Accessibility] Colour usage

### ISSUE THREAD

**Issue by zaccolley** [codebar/planner#583]
In some places the colour contrast of elements make it hard to read. WCAG guidelines a colour contrast ratio of at least 3:1 can read more here: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

For different sized text and weight the colour contrast required changes. Our body content for instance has a low weight so requires more contrast. We can up the font weight and get better readability without changing the colour.

Here's a list of place where the colour contrast doesn't meet this criteria:

1. [ ] Body content font weight means readiblity is low
2. [ ] Text in the footer
3. [ ] Upcoming / Attended links https://codebar.io/invitations
4. [ ] Alert box links and general colour
![image](https://user-images.githubusercontent.com/2591901/29586635-56613ae2-8783-11e7-956a-abfabe93d80e.png)
_While the text passes WCAG guidelines, reds and whites general pass but still offer a hard reading experience_
5. [ ] Highlighting in the menu
![image](https://user-images.githubusercontent.com/2591901/29586731-938f859a-8783-11e7-9fdb-13f99e2ee1bf.png)
6. [ ] Links in emails
![image](https://user-images.githubusercontent.com/2591901/29587836-1ec80de6-8787-11e7-85d1-ef4601e57b0a.png)
7. [ ] The logo: While logo text isnt a requirement from WCAG, the usage of the logo (on slides, temporary paper signage, other companies, the website, social media) means it needs to be as flexible as possible for readibility. Starting a base level of colour contrast (including line thickness) will give us the best chance of it still being readible when used in places we can't control.

**Comment by stphnnnn** [comment]
I spotted some of these while working on another issue yesterday, I feel this is quite an important change. 

I’ll take this on if it’s unclaimed...
